node_name: "hector_recorder"  
output_dir: "/home/user/bags"
#output_dir: "/path/to/output"      
storage_id: "sqlite3"              
#max_bag_size: 0           
#max_bag_size_gb: 1.0               # alternative to max_bag_size; use either this or max_bag_size not both
#max_bag_duration: 0             
#max_cache_size: 0          
#storage_preset_profile: "" 
#storage_config_uri: ""  
#snapshot_mode: false               
#start_time_ns: -1                  
#end_time_ns: -1                    
#custom_data:                       
  # "key1": "value1"
  # "key2": "value2"
all_topics: true
#all_topics: false                 
#all_services: false             
#is_discovery_disabled: false     
#topics:                         
  #- "/topic1"
  #- "/topic2"
#topic_types:                      
  #- "std_msgs/String"
#services:                         
  #- "/service1"
  #- "/service2"
#exclude_topics:                    
  #- "/excluded_topic"
#exclude_topic_types:               
  #- "std_msgs/Int32"
#exclude_service_events:            
  #- "/excluded_service_event"
rmw_serialization_format: "cdr"    
#topic_polling_interval: 100        
#regex: ""                      
#exclude_regex: ""         
#node_prefix: ""          
#compression_mode: ""           
#compression_format: ""        
#compression_queue_size: 0         
#compression_threads: 0         
#compression_threads_priority: 0    
#topic_qos_profile_overrides:       
  #"/topic1":
  #  history_depth: 10
  #  reliability: "reliable"
  #  durability: "volatile"
  #"/topic2":
  #  history_depth: 5
  #  reliability: "best_effort"
  #  durability: "transient_local"
#include_hidden_topics: false    
#include_unpublished_topics: false  
#ignore_leaf_topics: false         
#start_paused: false
#use_sim_time: false
#disable_keyboard_controls: false
#publish_status: true
#publish_status_topic: "recorder_status"  